spec/system/email_spec.rb

```diff
@@ -11,4 +11,17 @@
       expect(mail.body).to have_text('This is a GOV.UK Notify email with template')
     end
   end
+
```

Comment:
I've just noticed that this test seems to be for testing mailers and should probably live in `spec/mailers` instead. No fault of this PR though! It might be better to fix that separately?